package.json

```diff
@@ -99,6 +100,9 @@
     "flow-bin": "^0.70.0",
     "flow-coverage-report": "^0.5.0",
     "flow-typed": "^2.3.0",
+    "gitbook-cli": "^2.3.2",
+    "gitbook-plugin-ga": "1.0.1",
+    "gitbook-plugin-youtube": "1.1.0",
```

Comment:
As I understand it, we don't need the plugins here, as they're installed using npm in the `docs-user` subdirectory.

Don't forget to add yarn.lock after removing these lines and run `yarn install`

Reply:
An alternative solution is to move `book.json` to the root, use the `root` property to specify `docs-user`, but this isn't much better. I'd rather keep the current structure with the changes mentioned above.